recipes/oceanval/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.1.3" %}
+{% set python_min = "3.10" %}
+
+
+package:
+  name: oceanval
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/o/oceanval/oceanval-{{ version }}.tar.gz
+  sha256: 67384dc8b9fa9ceae6bdc0705e989bcc0c5554abf4a0c8aaf068d6936eea3bde 
+
+
+build:
+  number: 0
+  skip: True  # [win, osx]
```

Comment:
Looks like this one can be noarch.

It's okay if a noarch package fails on a specific platform due to a missing dependency. We prefer using noarch because it's faster to build, and the package will work as soon as the missing dependency becomes available on that platform.